### PR TITLE
Article Upgrade in 3 changes

### PR DESCRIPTION
**Pull request by pocc**
* Corrected grammar, syntax, and spelling throughput for readability
* Applied Prettier markdown formatting. File is now using markdown according to standards.
  Operation: `prettier --write TrueColour.md --prose-wrap always`
* Made 6 terminal status changes based on testing on my own Windows/Macos/Linux VMs

### CODE REVIEW

**Comment by XVilka** [comment]
Will check and merge soon, thanks.

**Comment by XVilka** [comment]
So far looks good - what about the ConEmu? It should support True Color for quite some time already: https://conemu.github.io/en/SettingsColors.html
Note the "[ ] TrueMod (24 bit color) support" check.

**Comment by pocc** [comment]
I do not believe that ConEmu support 24bit truecolor for these reasons:

* ConEmu does not claim it can support 24bit. It claims that it can support [xterm 256 colors](https://conemu.github.io/en/AnsiEscapeCodes.html#ANSI_sequences_processing_requirements).
* In my testing, ConEmu fails with `type colorline.ans` and `type perlansi.ans`, where colorline was generated with your AWK script and perlansi was generated by the [linked perl script](https://github.com/robertknight/konsole/blob/master/tests/color-spaces.pl). 

This is what I see: 

@Maximus5 is the maintainer of ConEmu, so please correct me if I'm wrong on support of 24bit color.

**Comment by Maximus5** [comment]
@pocc You are wrong.
ConEmu [docs states](https://conemu.github.io/en/AnsiEscapeCodes.html#SGR_Select_Graphic_Rendition_parameters) support of 24-bit colors via `ESC [ 38 ; 2 ; r ; g ; b m` and `ESC [ 48 ; 2 ; r ; g ; b m`. [Screenshot](https://conemu.github.io/en/Screenshots.html#ansi-xterm-256-and-24bit-colors-vim).
However, due to historical reasons and lack of a proper console API in Windows, the support of 8bit or 24bit colors are [limited to the bottom of the buffer](https://conemu.github.io/en/Xterm256Colors.html).

**Comment by pocc** [comment]
@Maximus5 Thank you for the feedback
@XVilka It looks like ConEmu deserves a longer (and separate) discussion, so I have readded it to the "Working" section for this pull request. Let me know if anything else needs changing.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-The 256 colour palete is configured at start, and it's a 666 cube of
-colours, each of them defined as a 24bit (888 rgb) colour.
+The 256-colour palette is configured at start and is a 666-cube of colours,
+each of them defined as a 24-bit (888 rgb) colour.
 
-This means that current support can only display 256 different colours
-in the terminal, while truecolour means that you can display 16 milion
-different colours at the same time.
+This means that current support can only display 256 different colours in the
+terminal while "TrueColour" means that you can display 16 million different
```
Comment (XVilka) [inline README.md]: I believe here it should be space in "'true colour"

Comment (pocc) [reply README.md]: Fixed, and also fixed #5. 

I think that `truecolor` the $COLORTERM setting should be one word, but when referring to the concept of 24bit color, use two words like "true color". I have made this change as part of the commit.

**Comment by XVilka** [comment]
Your last commit was totally wrong, I had to unroll it. Please check before committing.

**Comment by pocc** [comment]
Sorry about that. I was aiming for colour -> color with the last commit. I'll try again (and make sure no other changes seep in) as I still see 5 instances of the word "colours".

**Pull request merged**